samples/WebApp/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "webapp",
+  "version": "1.0.0",
+  "description": "Gluon sample web application",
+  "main": "Scripts/app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+
+    "clean-server": "xbuild /t:Clean /p:Configuration=Release ../SampleApp/SampleApp.fsproj",
+    "clean-webapp": "node_modules/.bin/rimraf ./bin && mkdir bin",
+    "clean": "npm run clean-server && npm run clean-webapp",
+
+    "build-server": "xbuild /t:Build /p:Configuration=Release ../SampleApp/SampleApp.fsproj",
```

Comment:
I would love to add `OutDir=bin` and have it dump the build output in `WebApp/bin` rather than having to copy the files. Unfortunately, I keep getting a permission issue.